roles/edpm_libvirt/molecule/default/prepare.yml

```diff
@@ -15,15 +15,28 @@
 # under the License.
 
 
-- name: Prepare
-  hosts: all
-  roles:
-    - role: ../../../../molecule/common/test_deps
-    - role: osp.edpm.env_data
+# - name: Prepare
+#   hosts: all
+#   roles:
+#     - role: ../../../../molecule/common/test_deps
+#     - role: osp.edpm.env_data
 
 - name: Setup DUT
   hosts: all
   tasks:
+    - name: Enable package repos
```

Comment:
I guess we need this in the cache role to allow caching the package

Reply:
so we only need this for ceph-common
its not available by default it seams on rocky/centos

alghouth in general on rhel its what would configure the downstream repo for our packages in general